PayloadPkg/FirmwareUpdate/FirmwareUpdate.c

```diff
@@ -893,6 +894,9 @@ ApplyFwImage (
       }
     }
     break;
+  case FW_UPDATE_COMP_CMD_UPDATE:
+    Status = FwCmdUpdateProcess (CapImage, CapImageSize);
+    break;

```

Comment:
should we call FwCmdUpdateProcess () with parameter ImageHdr to avoid search that image again?

Reply:
Sure Guo. I would check this.